package.json

```diff
@@ -34,19 +37,22 @@
   },
   "homepage": "https://github.com/prescottprue/tsheets-sdk",
   "devDependencies": {
-    "babel-cli": "^6.6.5",
-    "babel-core": "^6.7.2",
-    "babel-eslint": "^6.1.0",
+    "babel-cli": "^6",
```

Comment:
Was there a reason for making this less specific or did it just happen with an install?